design/010-new-dataset-fields/010-new-dataset-fields.md

```diff
@@ -0,0 +1,130 @@
+- Start Date: 22-07-2022
+- Status: Proposed
+
+# New metadata fields for datasets
+
+## Summary
+
+Add standard metadata fields to make Renku datasets more FAIR.
+
+## Motivation
+
+As a Renku user, I want to easily annotate my datasets with standard metadata. Renku datasets support custom annotations as JSON-ld files. Although these annotations are available in the project graph, they are not in the global KG. Furthermore, the process of adding custom metadata is tedious and error prone (requires writing JSON-ld by hand). Currently, some crucial default metadata fields (e.g. license) are missing from datasets.
+
+Datasets metadata should support common fields required for FAIR-ification.
+
+## Design Detail
+
+### User research data
+
+We can leverage information from 3 sources to identify potential fields. In order of decreasing priority: FAIR recommendations, User feedback, User persona.
+
+**FAIR recommendations**
+
+* F1: (Meta)data are assigned a globally unique and persistent identifier
+  + _Ideally we would use something like ROR for **affiliation**_
+* R1.1: data are released with a clear and accessible data usage **license**
+
+**Feedback**
+
+* I would like to know the total dataset **size** to adjust session ressources and avoid running out of disk space
+* Dataset imports are slow
+  + _A **size** attribute may help by showing an ETA or even “large dataset” warning in the future._
+* I would like to search entities by **affiliation** and **topic** in the search page
```

Comment:
is topic covered by keywords?

Reply:
That could work! This would require a special checkbox menu in the UI to select "topic" keywords among a set of possible values (like in dataverse). Alternatively, using autocomplete suggestions like for repository topics on github.